Sources/Plasma/NucleusLib/pnAsyncCoreExe/Pch.h

```diff
@@ -56,28 +56,21 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "plProduct.h"
 #include "hsThread.h"
-#include "hsWindows.h"
```

Comment:
I would leave this in, considering it's still being included in pnAceW32Thread

Reply:
Which is why I moved the include down to `pnAceW32Thread` instead of here globally...

Reply:
This project is goofy in that this is both the precompiled header and the global private header. What a mess. 😞